### DISCUSSION TITLE
Contributing, adding new cheats

### DISCUSSION THREAD

**Discussion by lasyan3** [Albeoris/Memoria#496] [General]
Hi,

I want to add new cheats as exp and ap multiplier.
I got the code, successfully compiled it and even able to multiply exp and ap.
But I read this in the "Developer instructions" section of the wiki:

> Please don't change any data that can be sent to the game server! We don't want any trouble.

I don't understand, what game server? What data? I don't want to make any trouble. Is it ok to pull requests of new cheats?
Thanks.

**Comment by Tirlititi** [comment]
Hello,

The datas sent to the game server are the different achievements (so you cannot create a new achievement) and mostly the game saves, that can be sent to the cloud.

However, it's not needed to do a pull request for a Exp / AP multiplier: Faospark's mod offers a wide variety of options for that already.

**Reply by snouz** [reply]
I didn't really know myself what that was, I added the examples to the wiki.

**Reply by lasyan3** [reply]
Oooh I didn't know about Faospark's mod, thanks.
